src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs

```diff
@@ -338,7 +357,33 @@ private ContactInformationDTO GetContacInfoDtoFromApplicant(MpdisApplicantDto ap
 
         private string GetItemDetailFromRequestModel(RequestModel requestModel)
         {
+            var tempWorkItem = GetByWorkItemById(requestModel.RequestID);
             WorkItemDetail itemDetail = new WorkItemDetail();
+
+            var tempWorkItemJson = JsonSerializer.Deserialize<WorkItemDetail>(tempWorkItem.Detail);
+
+
+            if (tempWorkItemJson.Status == null)
+            {
+                tempWorkItemJson.Status = new List<StatusHistoryItem>();
+            }
+            if (this.VerifyStatusDuplicate(tempWorkItemJson.Status.First(), requestModel))
+            {
+                //do nothing
+            }
+            else
+            {
+                tempWorkItemJson.Status.Insert(0, new StatusHistoryItem()
+                {
+                    ProcessingPhase = requestModel.ProcessingPhase,
+                    StatusText = requestModel.Status,
+                    ChangedBy = string.IsNullOrEmpty(requestModel.LoggedInUser) ? "Not assigned" : requestModel.LoggedInUser,
+                    RequestStatusTime = DateTime.UtcNow
+                }
+                );
+            }
```

Comment:
Avoid using empty if statement just reverse the if statement to remove the empty if.
```suggestion
            if (!this.VerifyStatusDuplicate(tempWorkItemJson.Status.First(), requestModel)) {
                tempWorkItemJson.Status.Insert(0, new StatusHistoryItem()
                {
                    ProcessingPhase = requestModel.ProcessingPhase,
                    StatusText = requestModel.Status,
                    ChangedBy = string.IsNullOrEmpty(requestModel.LoggedInUser) ? "Not assigned" : requestModel.LoggedInUser,
                    RequestStatusTime = DateTime.UtcNow
                }
                );
            }
```

Reply:
Changes have been made